spec/support/deser/property_de_ser.rb

```diff
@@ -29,3 +29,9 @@ class KeyChangePropertyDeSer < PropertyDeSer
 class CustomeSetterPropertyDeSer < PropertyDeSer
   property :string_prop, setter: :setter_string_prop
 end
+
+class OverriddenSetterPropertyDeSer < PropertyDeSer
+  def self.decimal_prop=(object, value)
+    object.decimal_prop = value.to_s
```

Comment:
Can you give me a more explicit example of what you are trying to do?

https://github.com/jsteinberg/ializer/blob/master/lib/ializer/string_de_ser.rb#L10